### PR TITLE
[honeyswap] Update network to xdai

### PR DESCRIPTION
**Pull request by youngkidwarrior**
It looks like the network change was overlooked. I changed it to xdai network

Changes proposed in this pull request:
- change network from `1` to `100`

### CODE REVIEW

**Comment by ChaituVR** [comment]
@youngkidwarrior would be great if you can add an address that can return a positive score for this strategy

**Comment by youngkidwarrior** [comment]
@ChaituVR done

**Comment by ChaituVR** [comment]
@youngkidwarrior Thanks but none of the addresses in examples.json return a positive score, would be great if you can add one/some addresses that can return some voting power

```
 "0xa478c2975ab1ea89e8196811f51a7b7ade33eb11",
      "0xeF8305E140ac520225DAf050e2f71d5fBcC543e7",
      "0x1E1A51E25f2816335cA436D65e9Af7694BE232ad",
      "0x1F717Ce8ff07597ee7c408b5623dF40AaAf1787C",
      "0x1c7a9275F2BD5a260A9c31069F77d53473b8ae2e",
      "0x1d5E65a087eBc3d03a294412E46CE5D6882969f4",
      "0x1f254336E5c46639A851b9CfC165697150a6c327",
      "0x2ec3F80BeDA63Ede96BA20375032CDD3aAfb3030",
      "0x4AcBcA6BE2f8D2540bBF4CA77E45dA0A4a095Fa2",
      "0x4F3D348a6D09837Ae7961B1E0cEe2cc118cec777",
      "0x6D7f23A509E212Ba7773EC1b2505d1A134f54fbe",
      "0x07a1f6fc89223c5ebD4e4ddaE89Ac97629856A0f",
      "0x8d5F05270da470e015b67Ab5042BDbE2D2FEFB48",
      "0x8d07D225a769b7Af3A923481E1FdF49180e6A265",
      "0x8f60501dE5b9b01F9EAf1214dbE1924aA97F7fd0",
      "0x9B8e8dD9151260c21CB6D7cc59067cd8DF306D58",
      "0x17ea92D6FfbAA1c7F6B117c1E9D0c88ABdc8b84C",
      "0x38C0039247A31F3939baE65e953612125cB88268"
      ```

**Comment by youngkidwarrior** [comment]
I see now! I'll get on this

File: src/strategies/honeyswap/index.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export const author = 'Kittyslasher';
 export const version = '0.1.0';
```
[the remark this answers is not in the document]
Comment (youngkidwarrior) [reply src/strategies/honeyswap/index.ts:5]: No proposals are using this atm

[the remark this answers is not in the document]
Comment (bonustrack) [reply src/strategies/honeyswap/index.ts:5]: Ok thanks

**Pull request merged**